src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -700,6 +705,12 @@ namespace PurchasedAirManager {
 
 			SetupOutputVariable( "Zone Ideal Loads Hybrid Ventilation Available Status []", PurchAir( PurchAirNum ).AvailStatus, "System", "Average", PurchAir( PurchAirNum ).Name );
 
+			//air flows
```

Comment:
When you look at this on github, line 706 is left of 704.  Doesn't seem correct.